src/semantics/ZExp.re

```diff
@@ -73,15 +73,15 @@ let rec make_inconsistent = (u_gen: MetaVarGen.t, ze: t): (t, MetaVarGen.t) =>
   | CursorE(cursor_side, e) =>
     let (e', u_gen) = UHExp.make_inconsistent(u_gen, e);
     (CursorE(cursor_side, e'), u_gen);
-  | Deeper(NotInHole, ze')
-  | Deeper(InHole(WrongLength, _), ze') =>
+  | Deeper(NotInHole, _)
+  | Deeper(InHole(WrongLength, _), _) =>
     let (u, u_gen) = MetaVarGen.next(u_gen);
     let ze' = set_err_status(InHole(TypeInconsistent, u), ze);
     (ze', u_gen);
   | Deeper(InHole(TypeInconsistent, _), _) => (ze, u_gen)
   | ParenthesizedZ(ze1) =>
     let (ze1', u_gen) = make_inconsistent(u_gen, ze1);
-    (ParenthesizedZ(ze1), u_gen);
+    (ParenthesizedZ(ze1'), u_gen);
```

Comment:
I think I fixed a bug here? Someone needs to confirm this.

Reply:
ah yeah I believe so